tests/test_utils.py

```diff
@@ -28,6 +28,20 @@ def test_does_not_add_path_if_it_does_not_exist(self):
         assert_that(self.not_existing_path in sys.path).is_false()
 
 
+class TestLoadModule(unittest.TestCase):
+    def setUp(self):
+        self.test_module_name = "os"
```

Comment:
@Chouvic One of the reasons I didn't write unit tests for this is because writting unit tests that actually load a file are kind of weak. You may endup having false negatives if something has loaded the module earlier.

I think you should remove these tests. The code is tested through the feature tests because they load definition files, and if they can't be loaded, as you saw, the tests fail.

Reply:
Yeah, that's correct. Tests removed.